src/Illuminate/Validation/Validator.php

```diff
@@ -962,7 +962,10 @@ protected function getSize($attribute, $value)
      */
     protected function validateIn($attribute, $value, $parameters)
     {
-        return in_array((string) $value, $parameters);
+        if(is_array($value))
+            return (count(array_diff($value, $parameters)) == 0);
+        else
+            return in_array((string) $value, $parameters);
```

Comment:
Please don't leave out the braces. It should be like this:

~~~php
if (is_array($value)) {
    return count(array_diff($value, $parameters)) == 0;
} else {
    return in_array((string) $value, $parameters);
}
~~~